### DISCUSSION TITLE
Manifest template

### DISCUSSION THREAD

**Discussion by Madis0** [Sinytra/Connector#613] [Ideas]
Let's assume I have a Fabric mod that works well on Forge and I want to let users know about it. 

Is there a way to have Forge-specific manifest file in the Fabric mod that'd tell them that they need this mod to use it? If so, can you make a template/tutorial of what's needed for it?

**Comment by Su5eD** [comment]
There's something I've seen Fabric modders do in this regard, and it works well with launchers, too: If you have your mod released on CurseForge or Modrinth, you can upload an additional release of your mod tagged for the Forge modloader and a dependency on Connector. For example, see [Eldritch End](https://legacy.curseforge.com/minecraft/mc-mods/eldritch-end/files/4932435). A complete list can be found [here](https://legacy.curseforge.com/minecraft/mc-mods/sinytra-connector/relations/dependents?filter-related-dependents=3).

**Reply by Madis0** [reply]
Well, you can mark the platform listings however you want, but many users use vanilla launcher, which does not enforce those requirements, nor download the dependencies for the user automatically.

**Comment by Su5eD** [comment]
This has now been implemented under the "Placeholder mods" feature. You can find all necessary information on our [docs page](https://sinytra.org/docs/connector/developers#placeholder-mods).
